pkg/install/convertconfig.go

```diff
@@ -100,8 +100,7 @@ func generateInstallConfig(cd *hivev1.ClusterDeployment, adminPassword, sshKey,
 		},
 		BaseDomain: spec.Config.BaseDomain,
 		Networking: types.Networking{
-			// TODO: installer currently only works with flannel, flagged for TODO on their end here: https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/installconfig.go#L82
-			Type: "flannel",
+			Type: "OpenshiftSDN",
```

Comment:
I hardcoded as this was all they supported at the time, now I think we can remove the hardcode, and instead use what convertNetworkPlugin returns, see the commented out code at the top of this function. Bring that back to life and then we can just pass through whatever the user specified.